core-rust/mesh-api-server/src/mesh_api/conversions/common.rs

```diff
@@ -0,0 +1,123 @@
+use crate::prelude::*;
+
+pub fn to_mesh_api_operation_no_fee(
```

Comment:
Let's call this `operations.rs` file instead of `common.rs`

Reply:
Sure